ROADMAP.md

```diff
@@ -132,3 +132,11 @@ for security reasons.
 
 Removal of these options from ProtocolConfiguration is scheduled for May-June
 2023 (~0.103.0 release).
+
+## GetPeers RPC server response type changes and rpcclient support
+
+GetPeers RPC command returns a list of Peers where the port type has changed from 
+string to uint16 to match C#. The rpcclient currently supports unmarshalling both
+formats. 
+
+Removal of Peer unmarshalling with string based ports is schedule for XXXdate (XXXrelease)
```

Comment:
@roman-khimov can you provide a date/version so I can update?

Reply:
~September 2023, ~0.105.0 release.